### ISSUE TITLE
Deploying the milestone 2 version of the project to amazon

### ISSUE THREAD

**Issue by GozdeUnver** [bounswe/2021SpringGroup5#360]
I will erase the previous version and deploy the milestone 2 version of the project

**Comment by GozdeUnver** [comment]
Currently, I deployed the longitude_latitude_post_updates branch to amazon because it works fine at my local. I deployed it and at first it gave db error so I deleted data/ file and deleted previous postgres image and then build the image again. Now it works fine

**Comment by GozdeUnver** [comment]
@salih997 and I deployed the master to the amazon. I added the CORS library in this [PR](https://github.com/bounswe/2021SpringGroup5/pull/404) so that frontend and mobile can send requests to backend. We deployed our backend code even though it is not finished totally because frontend and mobile needed to test their codes at this point and since the one on the amazon did not have CORS, they received error. It works fine now
